src/lib.rs

```diff
@@ -1,5 +1,15 @@
-pub mod bot;
+#![feature(async_closure)]
```

Comment:
Where is this feature used? I couldn't find any async code.

Reply:
I think I was trying something but it didn't work out.  Will remove